packages/salesforcedx-vscode-soql/package.json

```diff
@@ -35,6 +37,7 @@
     "typescript": "3.7.5"
   },
   "scripts": {
+    "build": "yarn compile",
```

Comment:
The repo uses `npm` not `yarn`

Reply:
That's because lerna, vsce (vscode cli used for publishing) and yarn don't play well yet.

Reply:
ok.  I'll adjust.

Reply:
habit